hoot-core/src/main/cpp/hoot/core/visitors/RemoveInvalidMultilineStringMembersVisitor.cpp

```diff
@@ -60,7 +60,7 @@ void RemoveInvalidMultilineStringMembersVisitor::visit(const ElementPtr &e)
         if (_map->getElement(it->getElementId())->getTags().getInformationCount() > 0)
           return;
       }
-      const double expansion = r->getCircularError();
+      const double expansion = r->getCircularError() / 2.0;
```

Comment:
Does this change sort of address this comment (by reducing the chance of extent intersections)?  https://github.com/ngageoint/hootenanny/pull/1527#issuecomment-297112508